src/view/treeNodes/pullRequestNode.ts

```diff
@@ -154,7 +154,20 @@ export class PRNode extends TreeNode implements CommentHandler, vscode.Commentin
 				this.pullRequestModel.githubRepository.commentsHandler!.clearCommentThreadCache(this.pullRequestModel.prNumber);
 			}
 
-			let result = [descriptionNode, ...this._fileChanges];
+			// group by directory
+			let dirNode = new DirectoryTreeNode(this, '');
+			this._fileChanges.forEach(f => dirNode.addFile(f));
+			dirNode.trimTree();
+			dirNode.sort();
```

Comment:
I think this may be unneeded, the files returned from the github API should already be sorted alphabetically